src/orchard/stacktrace/parser.clj

```diff
@@ -0,0 +1,42 @@
+(ns orchard.stacktrace.parser
```

Comment:
Each of the new namespaces should have some high-level description in its namespace docstring.

Reply:
Fixed.